docs/platform/release-notes/tizen-6-0-m2.md

```diff
@@ -0,0 +1,408 @@
+﻿# Tizen 6.0 Public M2
```

Comment:
One more comment :
With staging link https://docs1.stg.tizen.org/staging/1250/platform/release-notes/tizen-6-0-m2/,
we can see the first heading is not working well.

![image](https://user-images.githubusercontent.com/9958374/94383263-7b3a7a80-017a-11eb-9d92-6df8d2c053dc.png)

I doubt if there is a \<U+FEFF> in front of "\# Tizen 6.0 Public M2".
Would you please check and remove it?

Below screen shot is from the git history of previous release note.
![image](https://user-images.githubusercontent.com/9958374/94383506-38c56d80-017b-11eb-9ddb-9b7f20d5b279.png)

Reply:
I removed BOM as follows.
※ I'm not sure that <U+FEFF> is removed as it was not shown originally. So please check if <U+FEFF> has been removed.

 vi tizen-6-0-m2.md
 :set nobomb  // Remove BOM encoding:
 :wq  // Save and quit:

Reply:
The \<U+FEFF> is removed with the last update, so the heading is working well as below image. Thanks for the update. : )

![image](https://user-images.githubusercontent.com/9958374/95311243-50fa6280-08c8-11eb-8258-a71f81a194e1.png)